.github/workflows/terraform-compatibility-matrix.yml

```diff
@@ -0,0 +1,56 @@
+name: "HashiCorp Terraform Compatibility Matrix"
+run-name: 'HashiCorp Terraform Compatibility Matrix ${{ inputs.atlas_cloud_env }}'
+
+on:
+  schedule:
+    - cron: "0 0 1-7 * 6" # runs first Saturday of the month at midnight UTC
+  workflow_dispatch:
+    inputs:
+      atlas_cloud_env:
+        description: 'Atlas cloud environment used, can be either `dev` or `qa`, empty for `dev`'     
+        type: string
+        required: false
+      test_group:
+        description: 'Test group to run, e.g. advanced_cluster, empty for all'
+        type: string
+        required: false
+      ref:
+        description: 'The branch, tag or SHA where tests will run, e.g. v1.14.0, empty for default branch'
+        type: string
+        required: false
+
+jobs:
+  get-supported-versions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          ref: ${{ inputs.ref || github.ref }}
```

Comment:
I would remove inputs.ref as it's confusing in this GH action and it's not true the description: "The branch, tag or SHA where tests will run"

according to this code it's where the get-terraform-supported-versions script will run from.

I wouldn't do it at the moment,  but if we want to do it, we would have to change Test Suite to receive the param and pass to Acc Test GH action.

Reply:
Good point! I have removed ref completely. It will use the master branch now